src/Storage/Storage.Management/ChangeLog.md

```diff
@@ -18,6 +18,7 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
+* Upgrade to Storage Client Library 10.0.1 (the namespace of all objects from this SDK change from "Microsoft.WindowsAzure.Storage.*" to "Microsoft.Azure.Storage.*")
```

Comment:
It would likely be a good idea to point at a migration document here.

Reply:
How about we merge this first, and add the link later with another simple PR.